### ISSUE TITLE
RLS: 1.4

### ISSUE THREAD

**Issue by simonjayhawkins** [pandas-dev/pandas#41957]
Tracking issue for the 1.4 release. (it's possible we make this 2.0 at some point - not determined yet)

https://github.com/pandas-dev/pandas/milestone/86

Currently scheduled for December 31, 2021 (with a release candidate around 2 weeks before)

List of open regressions: https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+label%3ARegression

**Comment by simonjayhawkins** [comment]
@pandas-dev/pandas-core @pandas-dev/pandas-triage

**Comment by rhshadrach** [comment]
Is this a good place to discuss whether it should be 1.4/2.0? Or should a new issue be opened?

**Comment by simonjayhawkins** [comment]
here is fine imo. but not knowing how big a discussion it would be, opening a new issue is also fine by me.

**Comment by rhshadrach** [comment]
I'm +1 on 2.0

**Comment by jbrockmendel** [comment]
> whether it should be 1.4/2.0?

Has there been a discussion about unannounced API changes (e.g. deprecations that didn't make it into 1.3)?  I'm thinking of #41943 and some MultiIndex behavior that I would want to deprecate if we're doing a 1.4.

**Comment by jorisvandenbossche** [comment]
> some MultiIndex behavior that I would want to deprecate if we're doing a 1.4.

I think it is OK to do such deprecation if you want to look at that now, regardless of the final decision on 1.4 vs 2.0 (if it ends up becoming 2.0, the deprecation would then stay the full 2.x cycle I guess, like deprecations done in later 2.1 etc releases)

**Comment by rhshadrach** [comment]
Agree with @jorisvandenbossche - while we can make non-breaking changes in 2.0, I'd prefer sticking to a deprecation cycle (at least in general - perhaps a special case may be warranted). By deprecating, we allow user feedback before a removal/change takes place.

I'm changing my vote above, I'm +1 on 1.4 (but okay with 2.0 too). There are some apply deprecations I would like to get in so that things can start to be fixed in 2.0.

**Comment by jbrockmendel** [comment]
IIRC on last week's call we were leaning towards doing a 1.4

**Comment by lithomas1** [comment]
#42587 forces us to have a 1.4, since it changes the empty series deprecation warning from a DeprecationWarning to a FutureWarning. If we want to do 2.0, we'd have to revert that(not sure though if its ok to deprecate something with only a DeprecationWarning).

**Comment by auvipy** [comment]
> #42587 forces us to have a 1.4, since it changes the empty series deprecation warning from a DeprecationWarning to a FutureWarning. If we want to do 2.0, we'd have to revert that(not sure though if its ok to deprecate something with only a DeprecationWarning).

going with 1.4 would be a sane idea

**Comment by attack68** [comment]
I was hoping to raise a FutureWarning for the DataFrame.to_latex and DataFrame.to_html (and possibly DataFrame.to_string) methods to utilise the Styler implementation and to do away with the HTMLFormatter and LateX Formatter classes. That would be for 2.0, though, and currently those warnings would have to be in 1.4

**Comment by h-vetinari** [comment]
Regarding breaking changes, is there any appetite whatsoever to fix the return type of `Series.unique`, cf. https://github.com/pandas-dev/pandas/pull/24108? I didn't manage to get all thins into place before 1.0.

**Comment by jreback** [comment]
@h-vetinari yes that would be a change that would be good to make on a breaking release

**Comment by lithomas1** [comment]
On infra side, we should merge the following at MacPython/pandas-wheels(in the following order):

- [ ] MacPython/pandas-wheels#164 (needs whatsnew?)
- [ ] MacPython/pandas-wheels#160 
- [ ] MacPython/pandas-wheels#166 ( needs whatsnew? This needs rebase after MacPython/pandas-wheels#164)

cc @simonjayhawkins

**Comment by jreback** [comment]
@simonjayhawkins we can list any blocking issues here

iirc the only important one (which we may not have an issue for) is the pd.NumericIndex removal

cc @jorisvandenbossche 

otherwise i think ok to cut the RC soon

**Comment by jbrockmendel** [comment]
I'll spend some time tomorrow to see if there's any deprecations we should try to get in.

**Comment by simonjayhawkins** [comment]
> On infra side, we should merge the following at MacPython/pandas-wheels(in the following order):

we don't want to merge these if we need to do another 1.3.x release. It's been a week since 1.3.5 was released and if none of the issues raised so far warrant another release, we can start to think about merging those PRs.

**Comment by jbrockmendel** [comment]
Potential deprecations to get in for 1.4, checkboxes here mean "a decision has been made, one way or the other" (making a pass through the issues now, will update this list as I go)

- [ ] #44597 
- [ ] #40988
- [ ] #42921 (fixing the "bug" could constitute a breaking API change)

**Comment by auvipy** [comment]
numpy https://github.com/numpy/numpy/releases/tag/v1.22.0rc3 compat could be addressed to

**Comment by jreback** [comment]
> numpy https://github.com/numpy/numpy/releases/tag/v1.22.0rc3 compat could be addressed to

can u be more precise @auvipy

**Comment by simonjayhawkins** [comment]
> Potential deprecations to get in for 1.4

Thanks @jbrockmendel. can you also add the blocker for rc where appropriate (on issues and PRs) 

https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22Blocker+for+rc%22

Changes between the rc and the release should be minimal. ideally only issues that cropped up with the rc.

**Comment by jbrockmendel** [comment]
Marked #38240, #39584, #44353 as blockers for rc.  In each case we need to to come to a decision about the desired behavior.

**Comment by jreback** [comment]
there are simply too many open issues that are 'blocking' the rc: https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22Blocker+for+rc%22

so my suggestions are:
- release the rc now and then have a 1.5 where we actually try to address these issues
- wait x weeks to try to fix this where x <= 2
- kill a significant number of these and in 2 weeks release the rc.

cc @pandas-dev/pandas-core

**Comment by jbrockmendel** [comment]
> kill a significant number of these and in 2 weeks release the rc.

#44353 and #39584 _may_ be fine with no-action, but we _really_ should get the desired behavior pinned down.  I don't want to be stuck with undesired behavior for all of the 2.x series.

#38240 we have a game-plan in that I'm putting together a what-would-it-look-like branch.  ETA is "sometime this weekend"

#45038 is easy, i can Just Do It.

#43034 i think is an easy "no", though others will need to weigh in on that.

That just leaves #43206 and #41272 where I don't know what's going on.

**Comment by auvipy** [comment]
what about releasing alpha/beta 1 then rc first?

**Comment by jreback** [comment]
ok since we are seeing no movement here except for a single issue, then i am going to remove the blocker label for all the issues except: https://github.com/pandas-dev/pandas/issues/41272

and we'll just do a 1.5

**Comment by simonjayhawkins** [comment]
> and we'll just do a 1.5

for 1.4 we had decided to drop python 3.7 support https://github.com/pandas-dev/pandas/issues/41678#issue-902449213 so we could not release 1.4 (not rc) before Dec 26, 2021

so we could consider releasing 2.0 in less than 6 months after 1.5.

> what about releasing alpha/beta 1 then rc first?

we have nightly builds https://anaconda.org/scipy-wheels-nightly/pandas/files, so I'm not sure whether alphas/betas would add value.

**Comment by simonjayhawkins** [comment]
> so we could consider releasing 2.0 in less than 6 months after 1.5.

should have been... release 1.5 less than six months after 1.4.

**Comment by jbrockmendel** [comment]
un-marked #44353 and #39584 as blockers-for-RC.  Both should be high-ish priorities because they have many downstream implications, but neither needs to be done immediately.

**Comment by jreback** [comment]
ok i have moved everything off the blocker (to 1.5) except for https://github.com/pandas-dev/pandas/issues/41272

if someone wants to rebase and we can get in.

cc @simonjayhawkins

**Comment by jreback** [comment]
ok blocker list is all cleared /merged. @simonjayhawkins feel free to cut the RC!

**Comment by lithomas1** [comment]
I think this got buried in the comments but the infra PRs listed here https://github.com/pandas-dev/pandas/issues/41957#issuecomment-996194839 absolutely have to be merged before the RC. 
(Wheels will not build successfully without them).

**Comment by jreback** [comment]
good point @lithomas1

**Comment by jreback** [comment]
@simonjayhawkins i believe all the wheel builders have been merged

**Comment by simonjayhawkins** [comment]
have just completed a dummy release run locally on 3 machines.

long story short have been using wsl on a Windows machine for the releases up to now. The bottleneck has been the creation of the doc-build container. The lag between taging locally and pushing tag to ensure the docs build correctly.

Building the doc build image is done in parallel with the conda test and pip test. I don't have a dedicated internet connection and rely on the use of a WiFi hotspot (or tethering mobile). I think this may be partially responsible for some issues sometimes encountered.

I have changed the scripts to avoid some connection error issues and now use docker build kit to cache the conda packages but the conda solve time is still the bottleneck here (I think much longer following recent PR to use conda for more packages)

The machine used up to now is an 8 core i7-10700 with 64gb total (and 32gb available to wsl) but not sure how much is available to docker. But I think this is more than enough memory.

The other two machines are running ubuntu server over ssh. As well as evaluating performance, the idea being that the internet/issues maybe also be worse in windows although when doing dummy runs have not been able to replicate. The other reason is that if the release can be done over ssh then could maybe use a cloud instance instead (avoid internet connectivity issues altogether).

The 2 linux machines are a 6 core W-2135 (Lenovo workstation) and a dual 8 core E5-2667v2 (poweredge server). Performancewise, I don't think it makes a big difference since the bottleneck is the conda solve and seems to be similar duration on all three machines. (and speeding up tests (conda and pip) or the actual doc build with more cores makes little difference overall since these are done in parallel to the doc container build) 

So as this exercise has been inconclusive, I will continue to release using the existing Windows with wsl setup for now.

Will do some checks/review to get up to speed following the new year break and hopefully start the release later today/tomorrow.

**Comment by phofl** [comment]
You could use mamba to benchmark if conda is really the issue. Mamba should solve the enviroments near instantly if resources are sufficient

**Comment by simonjayhawkins** [comment]
> if conda is really the issue.

some timings...

`make docker-doc`

P340 3036.8s (2923.1)
P520 3536.6s (3370.8)
R620 3015.3s (2869.2)

These were executing in parallel with each other and the pip and conda tests, without an upto date cache)

with the packages cached and run sequentially on the 3 machines and without the test jobs running

P340 2302.4s
P520 2395.9s
R620 2629.3s

also a couple of `time conda env create`, creating conda env outside docker on the P520 for comparison

1st run 41m19.891s (2479.891s)
2nd run 31m12.997s (1932.997s)

**Comment by Dr-Irv** [comment]
Not sure if this is relevant, but I have docker running on Windows, which uses WSL under the hood, and found that the docker daemon starts eating memory on my machine as soon as you start creating images, start containers, etc.  (Note: I don't use docker to build pandas).  So that might be affecting performance.

Another thing to consider is your hard drive.  At the end of the day, a pandas build is going to hit your hard drive a lot, and if you don't have a solid-state drive, that could be the limiting factor in terms of performance.

**Comment by simonjayhawkins** [comment]
@Dr-Irv I noticed memory leaks with WSL and docker early last year but does not appear so much of any issue lately

the three machines have nvme drives. The conda solve step I assume is dependent on single core performance and internet connectivity. There is a 50% difference in single core perf of the i7-10700 and the e5-2667v2 so since the timings were not far adrift when the conda solve was the only job executing, I assume good internet connectivity is important.

**Comment by simonjayhawkins** [comment]
rc pre-release checklist

- [ ] MacPython.pandas nightly wheels built ok - https://dev.azure.com/pandas-dev/pandas-wheels/_build?definitionId=3&_a=summary
- [ ] release readiness passing https://github.com/simonjayhawkins/pandas-release/actions?query=workflow%3A%22Tag+Release%22
- [ ] release notes checked https://pandas.pydata.org/pandas-docs/dev/whatsnew/v1.4.0.html
- [ ] ci checks https://github.com/pandas-dev/pandas/commits/master
- [ ] no open PRs milestoned 1.4 that are not suitable backport candidates https://github.com/pandas-dev/pandas/pulls?q=is%3Aopen+is%3Apr+milestone%3A1.4
- [ ] no open issues milestoned 1.4 that are not suitable backport candidates https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.4
- [ ] no open blockers for rc https://github.com/pandas-dev/pandas/labels/Blocker%20for%20rc

**Comment by simonjayhawkins** [comment]
> * release notes checked https://pandas.pydata.org/pandas-docs/dev/whatsnew/v1.4.0.html

this can be tidied before release, but should probably include the contributors.

**Comment by simonjayhawkins** [comment]
> * MacPython.pandas nightly wheels built ok - https://dev.azure.com/pandas-dev/pandas-wheels/_build?definitionId=3&_a=summary

I think the failures are OK (i.e. flakey and could be restarted) but i'm not sure how to restart a failing job on the scheduled pipeline. We have not had a green nightly build since the recent PRs were merged so will open a "test status" PR to confirm.

**Comment by simonjayhawkins** [comment]
> but i'm not sure how to restart a failing job on the scheduled pipeline.

ahh. can be done from different screen (https://github.com/MacPython/pandas-wheels/runs/4713402432)

**Comment by simonjayhawkins** [comment]
we don't have any mention in the release notes about dropping Python 3.7. We should probably add this before the rc (but probably also in the GitHub release notes)

**Comment by simonjayhawkins** [comment]
@jreback Once I tag locally we can't merge to master till the `Start 1.5.0` tag has been pushed. Feel free to merge anything before I start the release but be aware that some PRs such as #45195 may require repeating some checks. (checks have been performed against c7d0d2775df9657fc362dcf418054d2394150972)

**Comment by jreback** [comment]
totally fine to wait-lmk when all is good

**Comment by jreback** [comment]
> we don't have any mention in the release notes about dropping Python 3.7. We should probably add this before the rc (but probably also in the GitHub release notes)

good point but can do after the rc is fine

**Comment by simonjayhawkins** [comment]
> * no open PRs milestoned 1.4 that are not suitable backport candidates https://github.com/pandas-dev/pandas/pulls?q=is%3Aopen+is%3Apr+milestone%3A1.4

bug fixes, regression fixes and further work on enhancements are ok to backport to 1.4.x. Not sure about #44451 and #45186. Can either move to 1.5 or maybe be less strict on these.

**Comment by simonjayhawkins** [comment]
> > we don't have any mention in the release notes about dropping Python 3.7. We should probably add this before the rc (but probably also in the GitHub release notes)
> 
> good point but can do after the rc is fine

#45197 is green. ok to get this in?

**Comment by simonjayhawkins** [comment]
> #45197 is green. ok to get this in?

ok. let's leave that for now.

@pandas-dev/pandas-core starting release now. please do not merge anything to master

- [ ] tag locally
- [ ] build sdist
- [ ] pip test
- [ ] conda test
- [ ] build docs
- [ ] push tag https://github.com/pandas-dev/pandas/tags
- [ ] create 1.4.x branch https://github.com/pandas-dev/pandas/tree/1.4.x
- [ ] start new tag v1.5.0.dev0 https://github.com/pandas-dev/pandas/releases/tag/v1.5.0.dev0
- [ ] activate ci on 1.4.x https://github.com/pandas-dev/pandas/pull/xxxxx
- [ ] open PR to build wheels https://github.com/MacPython/pandas-wheels/pull/xxx
- [ ] upload docs https://pandas.pydata.org/pandas-docs/version/1.4.0/
- [ ] wheels in staging area https://anaconda.org/multibuild-wheels-staging/pandas/files
- [ ] github release - https://github.com/pandas-dev/pandas/releases
- [ ] upload wheels to PyPI https://pypi.org/project/pandas/
- [ ] Start v1.5.0 release notes https://github.com/pandas-dev/pandas/pull/45198
- [ ] PR to conda-forge (conda-forge bot doesn't work for rc)
- [ ] available on conda-forge https://anaconda.org/conda-forge/pandas/files
- [ ] announce

**Comment by simonjayhawkins** [comment]
> @pandas-dev/pandas-core starting release now. please do not merge anything to master

can now merge to master. any issues with the release from now on in with be rectified with a rc1 on 1.4.x

**Comment by simonjayhawkins** [comment]
> * wheels in staging area https://anaconda.org/multibuild-wheels-staging/pandas/files

the macos universal wheels for 3.8 and 3.9 jobs added in https://github.com/MacPython/pandas-wheels/pull/167 appear to have the same filename for the wheel as the 64bit wheels so can't be sure which wheels were the last uploaded.  

I'll hold off the release pending further investigation/ green light from someone who knows better cc @lithomas1

**Comment by simonjayhawkins** [comment]
clocking off for today. If no further comments, will upload these wheels to PyPI tomorrow.

The issue is we have less wheels than MacPython jobs. IIUC we have 2 jobs that produce the same wheels as 2 other jobs, so I think no harm in moving forward and I don't think we need an rc1 to fix. Presumably, we need a PR to MacPython to fix and then just upload the 2 missing wheels to PyPI.

**Comment by lithomas1** [comment]
Thanks for catching my mistake. I've opened MacPython/pandas-wheels#169 to fix the issue with the universal2 wheels.

I think it should be fine to release without the aarch64 wheels, given that we provide an sdist and the conda-forge aarch64 package. I don't get why we ran out of credits though? I was under the assumption that they were provided under  Travis's Partner Queue Solution at a cost of zero credits.

In the future, we can also try moving to CircleCI on the wheel builder repo, but I don't have the permissions or time to do that.

**Comment by simonjayhawkins** [comment]
> I've opened [MacPython/pandas-wheels#169](https://github.com/MacPython/pandas-wheels/pull/169) to fix the issue with the universal2 wheels.

Thanks @lithomas1 for sorting that.

**Comment by simonjayhawkins** [comment]
> * available on conda-forge https://anaconda.org/conda-forge/pandas/files

done for 3.8 and 3.9. 3.10 outstanding.

**Comment by simonjayhawkins** [comment]
> 3.10 outstanding.

https://github.com/conda-forge/pandas-feedstock/pull/126

**Comment by simonjayhawkins** [comment]
> * available on conda-forge https://anaconda.org/conda-forge/pandas/files

The conda-forge packages have now been up for 24hrs and the PyPI wheels for a couple of days. As there have been no reports of installation issues, will announce shortly if no objections.

**Comment by simonjayhawkins** [comment]
> `Job has been terminated due to insufficient credits balance` on travis builds. This is the arm builds.

aarch64 wheels now uploaded to PyPI

**Comment by bashtage** [comment]
Might consider building aarch64 on GitHub using QEMU using cibuildwheel.  The big downside is the inability to run the full test suite.

**Comment by jmg-duarte** [comment]
What is the status on 1.4.0? I have a very specific use case which is fixed by 1.4.0 and I am curious to know a possible release date

**Comment by simonjayhawkins** [comment]
It was originally scheduled for 2 weeks after the rc date but since the wheels, conda packages and release announcement did not go out straight away, giving it a couple of extra days. So assuming no blockers, sometime in the next few days (probably Sunday)

**Comment by jreback** [comment]
@simonjayhawkins all PRs have been merged (pending 2 backports). I would move all issues off of 1.4

**Comment by simonjayhawkins** [comment]
There are a couple of things i'm about to look into:

from #45464 

> Also backport #45257 & #45323 since this PR sits on top of them and it makes sense to do so.

or how to get CI to green on 1.4.x (if it is something else)

also #45473 looks iffy to me at first glance with regards to the change in the na_rep keyword behavior (maybe just need some clarification in the docs)

**Comment by simonjayhawkins** [comment]
> I would move all issues off of 1.4

created 1.4.1 milestone https://github.com/pandas-dev/pandas/milestone/93 scheduled for February 13, 2022 (i.e. 3 weeks from Sunday) but date would be flexible on severity of any reported regressions

**Comment by simonjayhawkins** [comment]
> or how to get CI to green on 1.4.x (if it is something else)

the last green build was #45458 

actions-38-minimum_versions seems to be failing regularly on 1.4.x but with a different tests. The failure is a unexpected ResourceWarning that was occurring before we forked. This no longer occurs on master and AFAICT #45204 fixed so will need backport.

The other failures are timeouts which also occur on master. #45500 is investigating

> from #45464
> 
> > Also backport #45257 & #45323 since this PR sits on top of them and it makes sense to do so.

if not the cause of the CI failures does not strictly need backport but oth prob no harm in backporting those.

**Comment by simonjayhawkins** [comment]
release checklist

- [ ] wheels building https://github.com/MacPython/pandas-wheels/pull/173

restarted a run that failed on a flakey test. no reason to believe that won't pass on the restart


- [x] release readiness passing https://github.com/simonjayhawkins/pandas-release/actions?query=workflow%3A%22Tag+Release%22
- [ ] release notes checked https://pandas.pydata.org/pandas-docs/dev/whatsnew/v1.4.0.html

I will open a PR shortly to tidy these up and set the release date to tomorrow so that if we are in a good state to start the release tomorrow morning (UK time) we can release tomorrow.

I’ve not built the pdf locally (to check docs build correctly with release scripts) since #45370 (last dummy release performed earlier this week did include #45195). Will hopefully do that today.

- [ ] ci checks https://github.com/pandas-dev/pandas/commits/1.4.x

greenish

- [x] no manual backports outstanding https://github.com/pandas-dev/pandas/pulls?q=is%3Apr+label%3A%22Still+Needs+Manual+Backport%22+is%3Aclosed
- [ ] no open issues/PRs (except this one) https://github.com/pandas-dev/pandas/milestone/86

The 1.4.0 release will be short lived with a 1.4.1 release in around 3 weeks or less, so no reason to block on these. However, from https://github.com/pandas-dev/pandas/issues/45470#issuecomment-1016703054

> The fix should be relatively straightforward I think

So maybe could get that one sorted before release

**Comment by jorisvandenbossche** [comment]
@simonjayhawkins I would ask to wait a bit longer to release. Yesterday you said "probably Sunday", so I planned on that and am doing some fixes today for remaining regressions.

**Comment by simonjayhawkins** [comment]
Yeah let's aim for tomorrow and accept that if we are not ready we can do it Sunday. If we aim for Sunday, it may stretch into next week.

**Comment by jreback** [comment]
@jorisvandenbossche 

pls don't add anything else to 1.4

**Comment by simonjayhawkins** [comment]
release checklist

- [ ] wheels building https://github.com/MacPython/pandas-wheels/pull/173
- [ ] release readiness passing https://github.com/simonjayhawkins/pandas-release/actions?query=workflow%3A%22Tag+Release%22
- [ ] release notes checked https://pandas.pydata.org/pandas-docs/dev/whatsnew/v1.4.0.html
- [ ] ci checks https://github.com/pandas-dev/pandas/commits/1.4.x
- [ ] no manual backports outstanding https://github.com/pandas-dev/pandas/pulls?q=is%3Apr+label%3A%22Still+Needs+Manual+Backport%22+is%3Aclosed
- [ ] no open issues/PRs (except this one) https://github.com/pandas-dev/pandas/milestone/86

**Comment by simonjayhawkins** [comment]
> so that if we are in a good state to start the release tomorrow morning (UK time) we can release tomorrow.

starting release now

- [ ] tag locally
- [ ] build sdist
- [ ] pip test
- [ ] conda test
- [ ] build docs
- [ ] push tag https://github.com/pandas-dev/pandas/tags
- [ ] open PR to build wheels https://github.com/MacPython/pandas-wheels/pull/xxx
- [ ] upload docs https://pandas.pydata.org/pandas-docs/version/1.4.0/
- [ ] wheels in staging area https://anaconda.org/multibuild-wheels-staging/pandas/files
- [ ] github release - https://github.com/pandas-dev/pandas/releases
- [ ] upload wheels to PyPI https://pypi.org/project/pandas/
- [ ] change link for stable docs https://pandas.pydata.org/docs/
- [ ] start v1.4.1 release notes https://github.com/pandas-dev/pandas/pull/xxx
- [ ] conda-forge bot picked up on release https://github.com/conda-forge/pandas-feedstock/pull/xxx
- [ ] available on conda-forge https://anaconda.org/conda-forge/pandas/files
- [ ] announce

**Comment by simonjayhawkins** [comment]
> * conda-forge bot picked up on release https://github.com/conda-forge/pandas-feedstock/pull/xxx

have done this manually to also remove the 3.7 builds so will close the bot generated PR once opened

https://github.com/conda-forge/pandas-feedstock/pull/127

**Comment by rhshadrach** [comment]
For the new versions dropdown in the docs, I'm getting 1.3 (stable) takes me to 1.4, and the 1.4 dropdown is still showing "(rc)".

cc @jorisvandenbossche

**Comment by simonjayhawkins** [comment]
The website https://pandas.pydata.org/ still shows 1.3.5 as the latest release. This doesn't get updated until a merge to main after the release. Maybe the same for the docs dropdown?

**Comment by simonjayhawkins** [comment]
wheels and conda packages now available. will announce tomorrow if no serious installation issues reported.

**Comment by simonjayhawkins** [comment]
> For the new versions dropdown in the docs, I'm getting 1.3 (stable) takes me to 1.4, and the 1.4 dropdown is still showing "(rc)".

a brief look at #45370 and I guess web/pandas/versions.json needs to be updated manually

**Comment by jorisvandenbossche** [comment]
Thanks for the release, Simon!

> > For the new versions dropdown in the docs, I'm getting 1.3 (stable) takes me to 1.4, and the 1.4 dropdown is still showing "(rc)".
> 
> a brief look at #45370 and I guess web/pandas/versions.json needs to be updated manually

Yes, that's correct, and I see you already did that in the meantime (https://github.com/pandas-dev/pandas/pull/45555). We should probably add that somewhere to the release checklist

**Comment by lithomas1** [comment]
Can we close this issue now?
